### PR TITLE
feat(tabs): css changes

### PR DESCRIPTION
**Pull request by Barsnes**
part of #2295

### CODE REVIEW

File: packages/css/tabs.css [hunk 1]
```diff
@@ -23,59 +50,34 @@
...
-  --dsc-tabs__tab-bottom-border-color: var(--ds-color-accent-base-default);
-  --dsc-tabs-tab-color: var(--ds-color-accent-text-subtle);
-}
+  @composes ds-focus from './utilities.css';
 
-.ds-tabs__tab:focus {
-  z-index: 2;
-}
+  &:focus-visible {
+    z-index: 2;
```
Comment (eirikbacker) [inline packages/css/tabs.css:62]: Could we just do `position: relative` and also add a comment about why we are doing this? 😊

File: packages/css/tabs.css [hunk 2]
```diff
@@ -5,6 +5,33 @@
...
   --dsc-tabs-list-border-color: var(--ds-color-neutral-border-subtle);
+
+  &[data-size='sm'] {
+    --dsc-tabs-tab-padding: var(--ds-spacing-2) var(--ds-spacing-4);
+    --dsc-tabs-content-padding: var(--ds-spacing-4);
+  }
+
+  &[data-size='md'] {
+    --dsc-tabs-tab-padding: var(--ds-spacing-3) var(--ds-spacing-5);
+    --dsc-tabs-content-padding: var(--ds-spacing-5);
```
Comment (eirikbacker) [inline packages/css/tabs.css:16]: Should `md` be removed and just become part of default? ☺️

File: packages/css/tabs.css (hunk 1 above)
Comment (Barsnes) [reply packages/css/tabs.css:62]: No, we need to make sure it lands on top of the other buttons. Using just position relative makes the active line bleed through the focus indicator

File: packages/css/tabs.css (hunk 2 above)
Comment (Barsnes) [reply packages/css/tabs.css:16]: Yes, my bad

File: packages/css/tabs.css (hunk 1 above)
Comment (Barsnes) [reply packages/css/tabs.css:62]: This is what happens when we use just `position: relative`:
image

Comment (eirikbacker) [reply packages/css/tabs.css:57]: Right! comment on `z-index` and I'm happy <3

File: packages/css/tabs.css [hunk 3]
```diff
@@ -23,59 +45,34 @@
...
-.ds-tabs__tab[aria-selected='true'] {
-  --dsc-tabs__tab-bottom-border-color: var(--ds-color-accent-base-default);
-  --dsc-tabs-tab-color: var(--ds-color-accent-text-subtle);
-}
+  @composes ds-focus from './utilities.css';
 
-.ds-tabs__tab:focus {
-  z-index: 2;
-}
+  &:focus-visible {
```
Comment (Barsnes) [inline packages/css/tabs.css:56]:
```suggestion
  /* We set z-index to make sure the active line does not bleed over the focus indicator */
  &:focus-visible {
```

**Review by eirikbacker — APPROVED** [review]

**Pull request merged**